core/src/main/java/org/infinispan/transaction/xa/recovery/RecoveryManager.java

```diff
@@ -92,8 +101,8 @@
    String forceTransactionCompletionFromCluster(Xid xid, Address where, boolean commit);
 
    /**
-    * Checks both internal state and transaction table's state for the given tx. If it finds it, returns true if tx
-    * is prepared.
+    * Checks both internal state and transaction table's state for the given tx. If it finds it, returns true if tx is
+    * prepared.
```

Comment:
Hmmm, looks like GitHub actually wraps at 117 in the diff view, the old version looks better :)